sql/stripe/subscription_transitions/view.sql

```diff
@@ -0,0 +1,100 @@
+CREATE OR REPLACE VIEW
+  subscription_transitions
+AS
+WITH subscriptions AS (
+  SELECT
+    products_v1.name AS product,
+    (
+      cancel_at_period_end
+      OR "cancelled_for_customer_at" IN (SELECT key FROM UNNEST(subscriptions_v1.metadata))
+    ) AS cancelled_for_customer,
+    DATE(start_date) AS start_date,
+    DATE(trial_end) AS trial_end,
+    DATE(ended_at) AS end_date,
```

Comment:
so transitions like "new" occurs on the same date the subscription starts to be counted as `active`, and transitions like "cancelled" occur on the day after the last date subscription was counted as `active`